sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/aws/aws.yaml

```diff
@@ -0,0 +1,134 @@
+provider:
+  metadata:
+    name: AWS
+    category: Cloud provider
+    references:
+      - description: Best practices for managing AWS access keys
+        link: https://docs.aws.amazon.com/accounts/latest/reference/credentials-access-keys-best-practices.html
+        type: Documentation
+      - description: Managing access keys for IAM users
+        link: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html
+        type: Documentation
+      - description: OWASP Top 10 2021 Category A7 - Identification and Authentication Failures
+        link: https://owasp.org/Top10/A07_2021-Identification_and_Authentication_Failures/
+        type: Standards
+      - description: OWASP Top 10 2017 Category A3 - Sensitive Data Exposure
+        link: https://www.owasp.org/www-project-top-ten/2017/A3_2017-Sensitive_Data_Exposure
+        type: Standards
+      - description: MITRE, CWE-798 - Use of Hard-coded Credentials
+        link: https://cwe.mitre.org/data/definitions/798
+        type: Standards
+      - description: MITRE, CWE-259 - Use of Hard-coded Password
+        link: https://cwe.mitre.org/data/definitions/259
+        type: Standards
+      - description: SANS Top 25 - Porous Defenses
+        link: https://www.sans.org/top25-software-errors/#cat3
+        type: Standards
+  detection:
+    post:
+        patternNot: "EXAMPLE"
+  rules:
+    - id: aws-secret-access-key
+      metadata:
+        name: AWS Secret Access Keys should not be disclosed
+        message: Make sure this AWS Secret Access Key gets revoked, changed, and removed from the code.
+        impact: aws-secret-access.adoc
+      examples:
+        - text: >
+            var creds = new AWS.Credentials({
+                secretAccessKey: 'kHeUAwnSUizTWpSbyGAz4f+As5LshPIjvtpswqGb' 
+            });
+          containsSecret: true
+          match: kHeUAwnSUizTWpSbyGAz4f+As5LshPIjvtpswqGb
+        - text: >
+            aws_secret_access_key=kHeUAwnSUizTWpSbyGAz4f+As5LshPIjvtpswqGb
+          containsSecret: true
+          match: kHeUAwnSUizTWpSbyGAz4f+As5LshPIjvtpswqGb
+        - text: >
+            AWS_FECRET_KEY=kHeUAwnSUizTWpSbyGAz4f+As5LshPIjvtpswqGb'
+            AWS_SECRET_KEY=EXAMPLEKEYCXCgDCUbJq1h7CKwNqnpA1il4MXL+y
+          containsSecret: false
+      detection:
+        matching:
+          matchEach:
+            - pattern: "(?i)[0-9a-z\\/+]{40}"
+            - patternAround: "(?i)\\b(?:AWS)?_?SECRET_?(?:ACCESS)?_?KEY\\b"
```

Comment:
Same comment as in the other PR: for the first version it might be better to stick to the original regular expression, as this specification here is not the same and so it would probably change the results a bit.

For example, in the original specification, we only allowed a very tiny space between these keywords and the actual pattern. It was causing a lot of false positives otherwise.

The same applies to the other regular expressions in this file as well.